### PR TITLE
escalator: Fix escalators in north-point mall

### PR DESCRIPTION
**Pull request by JayFoxRox**
This is for #32, see header in file for what this does / how it works.

Code could be nicer, but this will do for now.

We could also make the stairs invisible, but I think this implementation also nicely hides the stairs appearing / disappearing.

### CODE REVIEW

File: scripts/module/escalator_fix.nut [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  local d = 1.924; // Distance between 2 stairs
+
+  // North point mall, as seen on radar, when facing north:
+  //
+  // 1   2
+  //   3
+  //   4
+  // 5   6
+  //
+  // Coordinates reflect the bottom of the left side
```
Comment (JayFoxRox) [inline scripts/module/escalator_fix.nut]: nit: Might not be true for all of them. See weird `d` / `-d` situation.

File: scripts/module/escalator_fix.nut [hunk 2]
```diff
@@ -0,0 +1,73 @@
+// VC-MP attempts to keep players standing on objects.
+// However, if a step of an escalator is warped for re-use, it basically takes
+// the player with it, accelerating them to high speeds.
+//
+// This script creates step models which make it impossible for the player
+// to stand on the real steps, thereby avoiding the issue.
+
+function create_stair_fix(world, p, a, count) {
```
Comment (JayFoxRox) [inline scripts/module/escalator_fix.nut]: `stair` -> `escalator`

File: scripts/module/escalator_fix.nut [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  local m = 584;
+  local l = 0.4;
+  local x = -l * sin(a);
+  local y =  l * cos(a);
+  for(local i = 0; i < count; i++) {
+    sfixb <- CreateObject( m, world, p + Vector(x * i, y * i, 0.0), 255);
+    sfixb.RotateToEuler(Vector(0,0,a), 0);
```
Comment (JayFoxRox) [inline scripts/module/escalator_fix.nut]: `sfixb` variable name is bad

**Comment by dracc** [comment]
Lines should end with `;`

File: scripts/module/escalator_fix.nut (hunk 1 above)
Comment (JayFoxRox) [reply scripts/module/escalator_fix.nut]: Comment added in code

File: scripts/module/escalator_fix.nut [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    o.RotateToEuler(Vector(0,0,a), 0);
+  }
+}
+
+function create_double_escalator_fix(world, p, a, count, d) {
+  create_escalator_fix(world, p, a, count);
+  create_escalator_fix(world, p + Vector(d, 0.0, 0.0), a, count);
+}
+
+function create_double_stair_up_down_fix(world, p, a, d) {
```
Comment (dracc) [inline scripts/module/escalator_fix.nut]: Still "stair"?

Comment (JayFoxRox) [reply scripts/module/escalator_fix.nut]: Meh.. I'll force push to master to fix this.

**Pull request merged**